src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -34,14 +41,20 @@ export interface ChatboxProps {
   displayPoll?: boolean;
 }
 
-export const _Chatbox: React.FC<ChatboxProps> = ({
+const NEXT_RENDER_SIZE = 50;
```

Comment:
please, move this to `settings`

Reply:
Moved & also renamed it to `CHATBOX_NEXT_RENDER_SIZE`.